README.md

```
@@ -31,249 +31,785 @@
 <br />
-<br />
-
-
-## 📋 Sobre o Projeto
-
-Sistema desenvolvido para **otimizar a gestão escolar da APAE**, oferecendo uma plataforma unificada para controle administrativo e pedagógico. A solução foi pensada para melhorar a organização, acessibilidade e eficiência dos processos diários da instituição.
 
-### ✨ Funcionalidades Principais
+## Sumário
+
+- [Sobre o Projeto](#-sobre-o-projeto)
```

markdownlint-cli2 (0.22.0)

[warning] Link fragments should be valid

(MD051, link-fragments)